seL4-doc/src/appendices/devkit_offline_use.md

```diff
@@ -88,6 +88,8 @@ In order to download source code for offline use, an internet-connected computer
     repo init -u https://github.com/sel4/sel4test-manifest.git
     ```
 
+Note that Git might request a username and email, which can be any string and does not have to be associated with a Git account.
```

Comment:
It's repo, not git, in this context. Also I suspect it's any string providing you just want a read-only copy?